recipes/zenoh-plugin-webserver/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "zenoh-plugin-webserver" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/eclipse-zenoh/zenoh-plugin-webserver/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 483cee2c6c15189219d663ff1353621b91941c10730e2381b724a3509b2917c5
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ stdlib('c') }}
+    - cargo-bundle-licenses
+    # required as on Linux unwind's rust functionality depends on gcc_s library
+    - {{ compiler('c') }}
+  host:
+    # for some reason  {{ version }}.{{ rust_compiler_version }} does not work
+    - zenoh-rust-abi {{ version }}.1.79.0
```

Comment:
Not sure what the error was but one likely thing could be that in the first pass over the recipe `conda-smithy` does not yet have a value for it. You could work around this by setting it to something at the top of the recipe if not set yet.

Reply:
Fixed in https://github.com/conda-forge/zenoh-plugin-webserver-feedstock/pull/1 .